extensions/nginx/acme.js

```diff
@@ -64,13 +74,19 @@ function generateCert(ui, domain, webroot, email, staging) {
 
         if (error.stderr.match(/Verify error:(Fetching|Invalid Response)/)) {
             // Domain verification failed
-            return Promise.reject(new cli.errors.SystemError(
-                'Your domain name is not pointing to the correct IP address of your server, please update it and run `ghost setup ssl` again'
-            ));
+            return Promise.reject(new cli.errors.SystemError({
```

Comment:
@acburdine I don't understand why we're using `SystemError` here, but further above `CliError`. What is the intention?